kobo/hub/models.py

```diff
@@ -731,6 +734,7 @@ def export(self, flat=True):
             "state_label": self.get_state_display(),
             "is_finished": self.is_finished(),
             "is_failed": self.is_failed(),
+            "retention_applied": self.retention_applied,
```

Comment:
Do you know what happens if the migration hasn't been applied, so that the DB column is missing?  Does the code return a default value or does it just crash?

I am concerned because this library adding migrations is a very rare event.  This means users of the library are not used to having to run a migration after upgrading kobo library and may not have incorporated that into their usual deployment process.